Makefile

```diff
@@ -0,0 +1,97 @@
+CONTAINER_NAME=splunk
+CONFIGURATION_BACKUP=$$(pwd)/.splunk_ta_dataset_config/
+OUTPUT_PACKAGE=$$(pwd)/output/TA_dataset
+SOURCE_PACKAGE=$$(pwd)/TA_dataset
+
+
+.PHONY: docker-run
+docker-run:
+	echo "$$(pwd)/output/TA_dataset:/opt/splunk/etc/apps/TA_dataset"
+	docker run -it \
+		-v "$(OUTPUT_PACKAGE):/opt/splunk/etc/apps/TA_dataset" \
```

Comment:
why do you echo `$(pwd)/output/TA_dataset` but using `$(OUTPUT_PACKAGE)` such an echo may not be very helpfull

Reply:
It's from the times where I was trying to figure out why is the volume empty in the docker. I have removed it.